pkg/kubelet/kubeletconfig/checkpoint/store/fsstore_test.go

```diff
@@ -39,38 +38,33 @@ import (
 	utilfs "k8s.io/kubernetes/pkg/util/filesystem"
 )
 
-var testdir string
-
-func init() {
-	tmp, err := ioutil.TempDir("", "fsstore-test")
-	if err != nil {
-		panic(err)
-	}
-	testdir = tmp
-}
-
 func newInitializedFakeFsStore() (*fsStore, error) {
 	// Test with the default filesystem, the fake filesystem has an issue caused by afero: https://github.com/spf13/afero/issues/141
 	// The default filesystem also behaves more like production, so we should probably not mock the filesystem for unit tests.
 	fs := utilfs.DefaultFs{}
 
-	tmpdir, err := fs.TempDir(testdir, "store-")
+	tmpDir, err := fs.TempDir("", "fsstore-test-")
 	if err != nil {
 		return nil, err
 	}
 
-	store := NewFsStore(fs, tmpdir)
+	store := NewFsStore(fs, tmpDir)
 	if err := store.Initialize(); err != nil {
 		return nil, err
 	}
 	return store.(*fsStore), nil
 }
 
+func removeAll(store *fsStore) {
```

Comment:
nit: I'm not sure removeAll is super informative in this context. Maybe cleanupFakeFsStore() to imply it should be paired with newInitializedFakeFsStore

Reply:
thanks, 
done!